### PR TITLE
nicola e niccolo

### PR DESCRIPTION
**Pull request by Nicolasprrt**
@nicrodemus

### CODE REVIEW

**Comment by Nicolasprrt** [comment]
We know, we should have used flex box! We were late and we used the position selector.

File: starter-code/apple-pie-starter-code/css/main.css [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    list-style-type: none;}
+
+header {
+        width: 100% ;
+        height: 500px;
+        background-image: url("../images/apple-pie.jpg"); 
+        background-size: cover;
+        }
+
+h1{ color: white;
```
Comment (ta-web-paris) [inline starter-code/apple-pie-starter-code/css/main.css]: an easier way to center your heading would be 
```
header {
display: flex
}

h1 {
align-items: center;
margin: auto;
}
```

File: starter-code/apple-pie-starter-code/css/main.css [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+}
+.intro  {
+    text-align: justify;
+    padding: 50px 350px 0px 350px;
+}
+
+
+.recipe-info img {
+    width: 300px;
```
Comment (ta-web-paris) [inline starter-code/apple-pie-starter-code/css/main.css]: to center an image you can use 

```
img {
display: block; 
margin: auto; 
}
```

then you don't need to used weird margins

**Comment by ta-web-paris** [comment]
good job! 

It looks good and you named all your classes well. Your code is well structured and easy to follow 💪

**Pull request closed without merging**